packaging/windows/host/host.wxs

```diff
@@ -35,9 +34,13 @@
         </Directory>
     </Fragment>
     <Fragment>
-        <ComponentGroup Id="InstallFiles">
+        <ComponentGroup Id="InstallSharedHostandDetectionKeys">
```

Comment:
Will this work in the following scenario on a clean machine?

1) Install 1.0.4
2) Install with fix
3) Uninstall 1.0.4

Does the new host remain? Also, what happens if, in (3), we uninstall host with the fix - does the old host remain?